myuw/static/js/handlebars-helpers.js

```diff
@@ -382,6 +382,17 @@ Handlebars.registerHelper('greater_than', function(value1, value2, options) {
     }
 });
 
+Handlebars.registerHelper('list_less_than', function(list, length, options) {
```

Comment:
I've removed this helper, as I found a built in handlebars feature that can do the same thing